packages/amplify-codegen-e2e-core/src/configure/index.ts

```diff
@@ -19,6 +19,7 @@ export const amplifyRegions = [
   'us-west-1',
   'us-west-2',
   'eu-north-1',
+  "eu-south-1",
```

Comment:
In Data repo, I noticed this change need to sync exactly with what's in cloudformation provider CLI package. Otherwise the select E2E helper selects a wrong region. Please make sure that this isn't the case with codegen.

Reply:
The new region is added in v10.8.0 and that is the point the e2e tests start to fail. As the codegen uses the latest cli-internal, this fix will be synced with the cfn provider

Reply:
Before this fix, the wrong region was picked when using CLI version based on 10.8.0